code/XMLTester.xml

```diff
@@ -1249,7 +1259,7 @@
 <builtin>
 <!-->XML-DURATION<-->
 <discovery_config>
-    <leaseDuration>INFINITE</leaseDuration> <!-- string -->
+    <leaseDuration><sec>DURATION_INFINITY</sec></leaseDuration> <!-- string -->
```

Comment:
Just to be consistent with the rest. Although not possible yet, it is going to be possible to do `<duration>DURATION_INFINITY<duration>`, which will also set the nanosecond to infinity, since like this they are set to 0 (not that it makes a difference to the user anyways).
```suggestion
    <leaseDuration>
        <sec>DURATION_INFINITY</sec> <!-- string -->
    </leaseDuration>
```

Reply:
When the new functionality is ready, we will create a follow-up PR on the docs with this.